controller/workitemtype_blackbox_test.go

```diff
@@ -98,13 +101,22 @@ func (s *workItemTypeSuite) SetupTest() {
 	require.NotNil(s.T(), s.spaceCtrl)
 }
 
+func (s *workItemTypeSuite) TearDownTest() {
+	s.clean()
+}
```

Comment:
We can by implementing the pattern in `DBTestSuite` directly:

```go
//...
type DBTestSuite struct {
	suite.Suite
	configFile string
	DB *gorm.DB
	clean func()
}
//...
func (s *DBTestSuite) SetupTest() {
	s.clean = cleaner.DeleteCreatedEntities(s.DB)
}
func (s *DBTestSuite) TearDownTest() {
	s.clean()
}
//...
```

Whenever you have to do something else in the `SetupTest()` just make sure you call the embedded function first:

```go
 func (s *myTestSuite) SetupTest() {
 	// Call embedded function first to setup the automatic cleaning.
 	s.DBTestSuite.SetupTest()

 	// Do what you have to do     
}
```